core/src/main/resources/reference.conf

```diff
@@ -659,6 +662,16 @@ cassandra-query-journal {
     # the offset to (offset + period). Larger values will result in a longer period of time
     # before the stream starts emitting events.
     offset-scanning-period = 200ms
+
+    # For eventsByTag queries how long to delay the query for. For event writes that come from different nodes
+    # the clocks may be out of sync meaning events aren't received in order. If the events are delivered to the
+    # query immediately the offset may be greater than some delayed events. Meaning that if this offset is saved
+    # for restarting the query the delayed events will never be processed.
+    eventual-consistency-delay = 10s
```

Comment:
I went with 10s as that what the old one was.

One case where it is always useful is when users save the offset for a restart. 
Even tho the new impl finds missed events when a delayed one is detected due to a tag/pid sequence nr gap it won't look before a provided offset. So if there is a crash/restart after some delayed events we'll restart with an offset after them.

Reply:
back then the delay played a more important role since we didn't have the tagPidSeqNr

I think 2 seconds would be acceptable for better global ordering, but if the purpose is also to reduce risk of missing events it might be another deal. If the risk is only when starting up a new query we could perhaps have a higher ec in the beginning and then reducing it?

Reply:
I went with 5s due to the case of resuming from an offset. I'd rather some one reduce it having read the doc then start with a small one and miss events on resume. WDYT?

Reply:
sounds good